apps/server/src/models/savings.ts

```diff
@@ -92,11 +91,6 @@ Savings.init(
       allowNull: false,
       defaultValue: DataTypes.DATE,
     },
-    created_at: {
-      type: DataTypes.DATE,
-      allowNull: false,
-      defaultValue: DataTypes.NOW,
-    },
```

Comment:
As you removed the createdAT, shouldn't you add timestamps to the table, or the timestamps are not needed? I think we should know when an action take place in the db for clear record sake. What do you think?